.github/workflows/bitcoin-tests.yml

```diff
@@ -1,205 +1,100 @@
-##
-## Bitcoin Integration Tests
-##
+## Github workflow to run bitcoin tests
 
-name: Bitcoin Integration Tests
-
-# Only run when:
-#   - PRs are (re)opened against master branch
+name: Tests::Bitcoin
 
 on:
-  pull_request:
-    types:
-      - opened
-      - reopened
+  workflow_call:
+
+defaults:
+  run:
+    shell: bash
```

Comment:
Is this necessary or just for clarity? Generally I tend to favor excluding explicit default settings to reduce LoC and perceived complexity (I know this isn't _technically_ the default, but it's very close to it). It's the ol' convention over configuration adage.

Reply:
it's technically not needed - but i went for verbosity here since github may change the runner environment at any time (today that `run:` would also work on ksh/zsh etc). 
i have no strong opinion here, the shell needs to be defined in the composite action, but since we're not calling `run:` here anymore, i'll remove it.